openshift_scalability/utils.py

```diff
@@ -350,7 +380,51 @@ def single_project(testconfig, projname, globalvars):
         pod_handler(testconfig["pods"], globalvars)
     if "rcs" in testconfig:
         rc_handler(testconfig["rcs"], globalvars)
-
+    if globalvars["autogen"]:
+        num_expected = int(globalvars["podnum"])
+        pods_running = []  
+        while len(pods_running) < num_expected: 
+            pods_running = subprocess.check_output(
+                "oc get pods --namespace=%s --no-headers |"
+                " awk '/1\/1/ && /Running/ {print $1;}'" \
+                % globalvars["namespace"], shell=True).splitlines()
+            time.sleep(5)
+
+        for pod in pods_running:
+            rsync = subprocess.check_output(
+                    "oc rsync --namespace=%s /root/.ssh %s:/root/" \
+                    % (globalvars["namespace"], pod), shell=True)
+
+        app = Flask(__name__)
+
+        @app.route("/")
+	def hello():
+	    return "GOTIME"
+
+	@app.route("/shutdown", methods=["POST"])
+	def shutdown_server():
+            func = request.environ.get("werkzeug.server.shutdown")
+            if func is None:
+                raise RuntimeError("Not running with the Werkzeug Server")
+            func()
+
+        def start_ws():
+            app.run(host="0.0.0.0", port=9090, threaded=True)
+
```

Comment:
I may be asking something stupid here, shouldn't have a rule, which actually tells to open this port..